### PR TITLE
add length to action return; make pagination total dynamic

### PR DESCRIPTION
**Pull request by chrispetito**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/actions/projectActions.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ export const getProjects = page => async dispatch => {
 
   try {
     const result = await axios.get(`${baseProjectsUrl}?page=${page}`)
-    dispatch({ type: GET_PROJECTS_SUCCESS, payload: result.data.projects })
+    // payload = projects from backend; projectLength = project length from backend for pagination total
```
Comment (lappjeff) [inline src/actions/projectActions.js]: Thanks for adding a clarifying comment here! 👾

File: src/components/ProjectSearch.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ import { getProjects } from "../actions"
 
 const ProjectSearch = props => {
   const [searchString, setSearchString] = useState("")
-  const { getProjects, projects } = props
+  const { getProjects, projects, projectLength } = props
+  // console.log(projectLength)
+  // console.log(projects)
```
Comment (lappjeff) [inline src/components/ProjectSearch.js]: Can we remove these commented out lines?

Comment (chrispetito) [reply src/components/ProjectSearch.js]: Removed!

**Comment by lappjeff** [comment]
This is great. Awesome work with the Redux flow and keeping the code clean and concise!

**Pull request merged**